trin-execution/src/main.rs

```diff
@@ -22,16 +23,14 @@ async fn main() -> anyhow::Result<()> {
         prometheus_exporter::start(addr)?;
     }
 
-    let directory = match trin_execution_config.ephemeral {
-        false => None,
-        true => Some(setup_temp_dir()?),
-    };
+    let data_dir = setup_data_dir(
```

Comment:
*nitpick* not sure if it's confusing that we accept an env var in `trin` but not here... Maybe it's worth adding a check just to see if the env var is set, and displaying a warning that it's not being used? Or maybe not...

Reply:
Trin is using `TRIN_DATA_PATH`. Should we do a check for that one (which doesn't make much sense), or for `TRIN_EXECUTION_DATA_PATH` (which doesn't exist and is never mentioned)? Or should I come up with new, shared, name?

I'm more in favor of completely removing support for env vars, but @carver made an argument that he finds it useful, so I'm leaving it. And if we are leaving that one, I'm not against adding another one (e.g. `TRIN_EXECUTION_DATA_PATH`), and have the same behavior. Thoughts?

Reply:
If we want to add an environment variable to `trin-execution` I think it should have a different variable name then for `trin`

> shared, name

I would not be a fan of ^. Because you might be running both on the same system.

In any case Trin will likely be used as a library in both
- portal bridge
- Trin Execution

One day (not fully related, just sharing what I think)

Reply:
Yeah, I'm also inclined more towards different name, if we want to add one.